src/sparkx/Storer/DummyStorer.py

```diff
@@ -0,0 +1,228 @@
+        are applied)
+
+
+    Methods
+    -------
+    particle_status:
+        Keep only particles with a given status flag
+    print_particle_lists_to_file:
+        Print current particle data to file with same format
+
+    Examples
+    --------
+    Create a Dummy object and access particle data as a nested list
+
+    >>> from sparkx.Particle import Particle
+    >>> particle1 = Particle()
+    >>> particle1.t = 1.0
+    >>> ...
+    >>> particle2 = Particle()
+    >>> particle2.t = 1.0
+    >>> ...
+    >>> particles = [particle1, particle2]
+    >>> dummy = DummyStorer(particles)
+    >>> nested_list = dummy.particle_list()
+    >>> print(nested_list)
+    [[[1.0,...],[1.0,...]]]
+
+    """
+    
+    def __init__(self, particle_object_list, **kwargs):
+        """
+        Initializes a new instance of the DummyStorer class.
+
+        This method initializes a new instance of the DummyStorer class with the specified list of particle objects and optional arguments. It calls the superclass's constructor with the particle_object_list and kwargs parameters and then deletes the loader_ attribute.
+
+        Parameters
+        ----------
+        particle_object_list : list of Particle objects
+            The list of particle objects to store.
+        kwargs : dict
+            A dictionary of optional arguments.
+
+        Raises
+        ------
+        None
+
+        Returns
+        -------
+        None
+        """
+        super().__init__(particle_object_list,**kwargs)  
+        del self.loader_
+
+    def create_loader(self, particle_object_list):
+        """
+        Creates a new DummyLoader object.
+
+        This method creates a new DummyLoader object with the specified list of particle objects and assigns it to the loader_ attribute.
+
+        Parameters
+        ----------
+        particle_object_list : list of Particle objects
+            The list of particle objects to load.
+
+        Raises
+        ------
+        None
+
+        Returns
+        -------
+        None
+        """
+        self.loader_= DummyLoader(particle_object_list)
+
+    def _particle_as_list(self, particle):
+        """
+        Converts a Particle object into a list.
+
+        This method takes a Particle object and converts it into a list of its attributes. The attributes are added to the list in the following order: t, x, y, z, mass, E, px, py, pz, pdg, ID, charge, ncoll, form_time, xsecfac, proc_id_origin, proc_type_origin, t_last_coll, pdg_mother1, pdg_mother2, baryon_number, strangeness, weight, status.
+
+        Parameters
+        ----------
+        particle : Particle
+            The Particle object to convert into a list.
+
+        Raises
+        ------
+        None
+
+        Returns
+        -------
+        particle_list : list
+            A list of the attributes of the Particle object.
+        """
+        particle_list = []
+        particle_list.append(float(particle.t))
```

Comment:
Are the casts here necessary? I think in the Particle class we return the correct types if there are integers.

Reply:
Where do I find the corresponding function for Jetscape?